validphys2/src/validphys/fkparser.py

```diff
@@ -312,13 +313,18 @@ def parse_fktable(f):
             hadronic = res['GridInfo'].hadronic
             ndata = res['GridInfo'].ndata
             xgrid = res.pop('xGrid')
+            # There are no non-unpolarized FK tables with the old theory,
+            # so everything is `UnpolPDF`.
+            nb_convolutions = 2 if hadronic else 1
+            conv_types = tuple("UnpolPDF" for _ in range(nb_convolutions))
```

Comment:
A nitpick, but I dislike intermediate variables that just hide what's really happening.

```suggestion
            if hadronic:
                conv_types = ("UnpolPDF", "UnpolPDF")
            else:
                 conv_types = ("UnpolPDF",)
```